lighthouse-cli/test/smokehouse/test-definitions/timing.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @type {LH.Config.Json} */
+const config = {
+  extends: 'lighthouse:default',
+  settings: {
+    onlyAudits: ['viewport'],
+  },
+};
+
+/**
+ * @type {Smokehouse.ExpectedRunnerResult}
+ */
+const expectations = {
+  artifacts: {
+    Timing: {
+      3: {
+        name: 'lh:runner:gather',
+      },
+    },
+  },
+  lhr: {
+    requestedUrl: 'http://localhost:10200/simple-page.html',
+    finalUrl: 'http://localhost:10200/simple-page.html',
+    audits: {},
+    timing: {
+      // TODO: Check for existence of lh:runner:audit.
+      // The exact position of lh:runner:audit can vary depending on runner.
```

Comment:
Depending on the runner (fraggle rock, bundle, cli) these timings can be in different orders or named differently. `lh:runner:gather` and `lh:runner:audit` should always be present, but only `lh:runner:gather` happens to be at the same index every time.